CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [3.0.2] - TBD
 ### Fixed
 - Set lifecycle type to `UNREFERENCED` for paths picked up by Vacuum tool. 
+### Changed
```

Comment:
I think we don't put README updates in CHANGELOG.

Reply:
fair enough